### PR TITLE
[4.0] Add the new doc page to the HTTP Headers Plugin postinstall message

### PR DESCRIPTION
**Pull request by zero-24**
### Summary of Changes

- Add the new doc page to the HTTP Headers Plugin postinstall message
- add other Explanation links.

### Testing Instructions

- Install 4.0
- disable the plg_system_httpheaders Plugin
- check the postinstall

### Expected result

image

### Actual result

image

### Documentation Changes Required

none

### CODE REVIEW

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on fa20f96b204a54cbec740dee8c16fab2dc623cc1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24687">issues.joomla.org/tracker/joomla-cms/24687</a>.</sub>

File: administrator/language/en-GB/en-GB.plg_system_httpheaders.ini [hunk 1]
```diff
@@ -29,7 +29,7 @@ PLG_SYSTEM_HTTPHEADERS_HSTS_SUBDOMAINS_DESC="HSTS should also be enabled <strong
...
...blank' rel='noopener noreferrer'>X-XSS-Protection</a></li><li><a href='https://scotthelme.co.uk/hardening-your-http-response-headers/#x-content-type-options' target='_blank' rel='noopener noreferrer'>X-Content-Type-Options</a></li><li><a href='https://scotthelme.co.uk/a-new-security-header-referrer-policy/' target='_blank' rel='noopener noreferrer'>Referrer-Policy</a></li><li><a href='https://scotthelme.co.uk/a-new-security-header-expect-ct/' target='_blank' rel='noopener noreferrer'>Expect-CT</a></li><li><a href='https://scotthelme.co.uk/a-new-security-header-feature-policy/' target='_blank' rel='noopener noreferrer'>Feature-Policy</a></li></ul><br>These headers help your browser for example to protect your website from <a href='https://en.wikipedia.org/wiki/Cross-site_scripting' target='_blank' rel='noopener noreferrer'>XSS</a> and <a href='https://en.wikipedia.org/wiki/Clickjacking' target='_blank' rel='noopener noreferrer'>Clickjacking</a> attacks.<br>You can find more details in the <a href='https://docs.joomla.org/Special:MyLanguage/J4.x:Http_Header_Management' target='_blank' rel='noopener noreferrer'>HTTP Header Management Tutorial in the Joomla! Documentation.</a><br><br>"
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.plg_system_httpheaders.ini]:
```suggestion
PLG_SYSTEM_HTTPHEADERS_POSTINSTALL_INTRODUCTION_BODY="<p>Joomla! comes with a built-in set of tools that help you to handle http security headers. It helps to secure your site by setting the following headers with the default values:</p><ul><li><a href='https://scotthelme.co.uk/hardening-your-http-response-headers/#x-frame-options' target='_blank' rel='noopener noreferrer'>'X-Frame-Options: SAMEORIGIN'</a></li><li><a href='https://scotthelme.co.uk/hardening-your-http-response-headers/#x-xss-protection' target='_blank' rel='noopener noreferrer'>'X-XSS-Protection: 1; mode=block'</a></li><li><a href='https://scotthelme.co.uk/hardening-your-http-response-headers/#x-content-type-options' target='_blank' rel='noopener noreferrer'>'X-Content-Type-Options: nosniff'</a></li><li><a href='https://scotthelme.co.uk/a-new-security-header-referrer-policy/' target='_blank' rel='noopener noreferrer'>'Referrer-Policy: no-referrer-when-downgrade'</a></li></ul><p>The full list of supported headers are:</p><ul><li><a href='https://scotthelme.co.uk/hsts-the-missing-link-in-tls/' target='_blank' rel='noopener noreferrer'>Strict-Transport-Security</a></li><li><a href='https://scotthelme.co.uk/content-security-policy-an-introduction/' target='_blank' rel='noopener noreferrer'>Content-Security-Policy</a></li><li><a href='https://scotthelme.co.uk/content-security-policy-an-introduction/#testingapolicy' target='_blank' rel='noopener noreferrer'>Content-Security-Policy-Report-Only</a></li><li><a href='https://scotthelme.co.uk/hardening-your-http-response-headers/#x-frame-options' target='_blank' rel='noopener noreferrer'>X-Frame-Options</a></li><li><a href='https://scotthelme.co.uk/hardening-your-http-response-headers/#x-xss-protection' target='_blank' rel='noopener noreferrer'>X-XSS-Protection</a></li><li><a href='https://scotthelme.co.uk/hardening-your-http-response-headers/#x-content-type-options' target='_blank' rel='noopener noreferrer'>X-Content-Type-Options</a></li><li><a href='https://scotthelme.co.uk/a-new-security-header-referrer-policy/' target='_blank' rel='noopener noreferrer'>Referrer-Policy</a></li><li><a href='https://scotthelme.co.uk/a-new-security-header-expect-ct/' target='_blank' rel='noopener noreferrer'>Expect-CT</a></li><li><a href='https://scotthelme.co.uk/a-new-security-header-feature-policy/' target='_blank' rel='noopener noreferrer'>Feature-Policy</a></li></ul><p>These headers help your browser for example to protect your website from <a href='https://en.wikipedia.org/wiki/Cross-site_scripting' target='_blank' rel='noopener noreferrer'>XSS</a> and <a href='https://en.wikipedia.org/wiki/Clickjacking' target='_blank' rel='noopener noreferrer'>Clickjacking</a> attacks.</p><p>You can find more details in the <a href='https://docs.joomla.org/Special:MyLanguage/J4.x:Http_Header_Management' target='_blank' rel='noopener noreferrer'>HTTP Header Management Tutorial in the Joomla! Documentation.</a></p>"
```

**Comment by brianteeman** [comment]
There must be a better way of creating this message. It's going to be almost impossible for the translators to work out what to translate and what to ignore.

**Comment by Quy** [comment]
Good point. Lets simplify it with detailed information in the doc.

>Joomla! comes with a built-in set of tools that help you to handle http security headers. These headers help your browser for example to protect your website from XSS and Clickjacking attacks.
>
>You can find more details in the HTTP Header Management Tutorial in the Joomla! Documentation.

**Comment by zero-24** [comment]
Implemented the suggestion cc @Quy @brianteeman 

image

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 834e68aad481105401829d14259d7a33ac30ef51<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24687">issues.joomla.org/tracker/joomla-cms/24687</a>.</sub>

**Comment by sanderpotjer** [comment]
I have tested this item :white_check_mark: successfully on 834e68aad481105401829d14259d7a33ac30ef51<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24687">issues.joomla.org/tracker/joomla-cms/24687</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24687">issues.joomla.org/tracker/joomla-cms/24687</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**